src/position.rs

```diff
@@ -9,6 +9,8 @@ pub struct AprsPosition {
     pub timestamp: Option<Timestamp>,
     pub latitude: f32,
     pub longitude: f32,
+    pub symbol_table: char,
+    pub symbol_code: char,
```

Comment:
I'm wondering if it would make sense to use an `enum` for these 🤔

Reply:
I tried out...